### PR TITLE
Regression: Added missing call button in the contact info contextual bar

### PR DESCRIPTION
**Pull request by aleksandernsilva**

### CODE REVIEW

File: apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactInfo.js [hunk 1]
```diff
@@ -157,13 +163,20 @@ const ContactInfo = ({ id, rid, route }) => {
...
+				<ButtonGroup stretch flexWrap='wrap'>
+					{isCallReady && (
+						<>
+							<VoipInfoCallButton phoneNumber={phoneNumber} mi={0} flexBasis='0' />
+							{showContactHistory && <Divider width='100%' />}
+						</>
+					)}
+
+					{showContactHistory && (
+						<Button onClick={onChatHistory} mis={0} flexBasis='0'>
```
Comment (aleksandernsilva) [inline apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactInfo.js:175]: This logic is necessary, so we can have both layouts depending on the number of buttons:
Screen Shot 2022-07-05 at 13 03 22
Screen Shot 2022-07-05 at 13 03 33

Comment (tiagoevanp) [reply apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactInfo.js:175]: Wait, this divider is not aligned... can you fix this?

**Review by tiagoevanp — CHANGES_REQUESTED** [review]

File: apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactInfo.js (hunk 1 above)
Comment (aleksandernsilva) [reply apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactInfo.js:175]: Yeah, this image is outdated. I reused an old print just to give the idea 😅 
Everything should be aligned.

**Review by tiagoevanp — APPROVED** [review]

**Review by ggazzo — CHANGES_REQUESTED** [review]
already dm alek about how to improve

**Pull request merged**